documentation/modules/deploying/proc-deploy-topic-operator-with-cluster-operator.adoc

```diff
@@ -9,7 +9,10 @@
 This procedure describes how to deploy the Topic Operator using the Cluster Operator.
 
 You configure the `entityOperator` property of the `Kafka` resource to include the `topicOperator`.
-By default, the Topic Operator watches for `KafkaTopics` in the namespace of the Kafka cluster deployment.
+By default, the Topic Operator watches for `KafkaTopic` resources in the namespace of the Kafka cluster deployed by the Cluster Operator.
+You can also specify a namespace using `watchedNamespace` in the Topic Operator `spec`.
+The Topic Operator can only watch a single Kafka cluster in a namespace.
+If you want to use the operator with more than one Kafka cluster, install the clusters to separate namespaces.
```

Comment:
```suggestion
The Topic Operator can only watch a single namespace.
And a single namespace should be watched only by a single Topic Operator.
```

Reply:
Can we just say: _A single Topic Operator can watch a single namespace._